test/layers/conv.jl

```diff
@@ -16,387 +16,387 @@ adj_single_vertex =   T[0. 0. 0. 1.;
-            @test graph(ggc.fg) == [[2,4], [1,3], [2,4], [1,3]]
-            @test size(ggc.weight) == (out_channel, out_channel, num_layers)
-
-            Y = ggc(X)
-            @test size(Y) == (out_channel, N)
-
-
-            # Test with transposed features
-            Y = ggc(Xt)
-            @test size(Y) == (out_channel, N)
-
-            g = Zygote.gradient(x -> sum(ggc(x)), X)[1]
-            @test size(g) == size(X)
-
-            g = Zygote.gradient(model -> sum(model(X)), ggc)[1]
-            @test size(g.weight) == size(ggc.weight)
-        end
-
-        @testset "layer without graph" begin
-            ggc = GatedGraphConv(out_channel, num_layers)
-            @test size(ggc.weight) == (out_channel, out_channel, num_layers)
-
-            fg = FeaturedGraph(adj, nf=X)
-            fg_ = ggc(fg)
-            @test size(node_feature(fg_)) == (out_channel, N)
-            @test_throws AssertionError ggc(X)
-
-
-            # Test with transposed features
-            fgt = FeaturedGraph(adj, nf=Xt)
-            fgt_ = ggc(fgt)
-            @test size(node_feature(fgt_)) == (out_channel, N)
-
-            g = Zygote.gradient(x -> sum(node_feature(ggc(x))), fg)[1]
-            @test size(g[].nf) == size(X)
-
-            g = Zygote.gradient(model -> sum(node_feature(model(fg))), ggc)[1]
-            @test size(g.weight) == size(ggc.weight)
-        end
-    end
-
-    @testset "EdgeConv" begin
-        X = rand(Float32, in_channel, N)
-        Xt = transpose(rand(Float32, N, in_channel))
-        @testset "layer with graph" begin
-            ec = EdgeConv(adj, Dense(2*in_channel, out_channel))
-            @test graph(ec.fg) == [[2,4], [1,3], [2,4], [1,3]]
-
-            Y = ec(X)
-            @test size(Y) == (out_channel, N)
-
-
-            # Test with transposed features
-            Y = ec(Xt)
-            @test size(Y) == (out_channel, N)
-
-            g = Zygote.gradient(x -> sum(ec(x)), X)[1]
-            @test size(g) == size(X)
-
-            g = Zygote.gradient(model -> sum(model(X)), ec)[1]
-            @test size(g.nn.W) == size(ec.nn.W)
-            @test size(g.nn.b) == size(ec.nn.b)
-        end
-
-        @testset "layer without graph" begin
-            ec = EdgeConv(Dense(2*in_channel, out_channel))
-
-            fg = FeaturedGraph(adj, nf=X)
-            fg_ = ec(fg)
-            @test size(node_feature(fg_)) == (out_channel, N)
-            @test_throws AssertionError ec(X)
-
-
-            # Test with transposed features
-            fgt = FeaturedGraph(adj, nf=Xt)
-            fgt_ = ec(fgt)
-            @test size(node_feature(fgt_)) == (out_channel, N)
-
-            g = Zygote.gradient(x -> sum(node_feature(ec(x))), fg)[1]
-            @test size(g[].nf) == size(X)
-
-            g = Zygote.gradient(model -> sum(node_feature(model(fg))), ec)[1]
-            @test size(g.nn.W) == size(ec.nn.W)
-            @test size(g.nn.b) == size(ec.nn.b)
-        end
-    end
+     @testset "GCNConv" begin
```

Comment:
Please don't change the code unrelated to this PR.

Reply:
Oops! I reverted it back.